src/MeshVolume.js

```diff
@@ -171,6 +182,60 @@ export default class MeshVolume {
     return this.meshrep[channel].userData.isovalue;
   }
 
+  updateClipRegion(xmin, xmax, ymin, ymax, zmin, zmax) {
+    this.bounds = {
+      bmin: new Vector3(xmin - 0.5, ymin - 0.5, zmin - 0.5),
+      bmax: new Vector3(xmax - 0.5, ymax - 0.5, zmax - 0.5),
+    };
+    this.updateClipFromBounds();
+  }
+
+  updateClipFromBounds() {
+    const xmin = this.bounds.bmin.x;
+    const ymin = this.bounds.bmin.y;
+    const zmin = this.bounds.bmin.z;
+    const xmax = this.bounds.bmax.x;
+    const ymax = this.bounds.bmax.y;
+    const zmax = this.bounds.bmax.z;
+
+    const euler = this.meshPivot.rotation;
+
+    // assuming these are 0..1
```

Comment:
nit I think, you mean 0.1 here right?

Reply:
no, i am using `0..1` to mean in the range from 0 to 1. i can change the comment to make that more clear

Reply:
ooooh, yeah `0 <= x <= 1` would make more sense to me, or just `between 0 and 1, inclusive`

Reply:
ugh, the comment makes no sense where it is, anyway.  It's meant to refer to the bounding box extent values.

Reply:
The new comment you just added makes a lot of sense!